File: corporate-reporting/src/types/RiskRiskType.ts
import { IEntity, Entity } from "./Entity";
import { IRiskType } from "./RiskType";

export interface IRiskRiskType extends IEntity {
    RiskID: number;
    RiskTypeID: number;
    RiskType?: IRiskType;
}

export class RiskRiskType extends Entity implements IRiskRiskType {
    public RiskID = null;
    public RiskTypeID = null;

}

File: ORB.Core/Models/RiskRiskType.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class RiskRiskType : Entity
    {
        public int RiskID { get; set; }
        public int RiskTypeID { get; set; }

        public virtual CorporateRisk CorporateRisk { get; set; }
        public virtual RiskType RiskType { get; set; }
    }
}
